keras_nlp/tokenizers/sentence_piece_tokenizer.py

```diff
@@ -159,7 +159,7 @@ def vocabulary_size(self) -> int:
         return int(self._sentence_piece.vocab_size().numpy())
 
     def get_vocabulary(self) -> List[str]:
-        """Get the size of the tokenizer vocabulary."""
+        """Get the tokenizer vocabulary."""
```

Comment:
Is there any downside to making the super class impl use `self._sentence_piece.vocab_size()` instead of `self.vocabulary_size()` here? Then we don't need all this indirection on the subclass.